kobo/apps/organizations/models.py

```diff
@@ -12,6 +12,16 @@
 class Organization(AbstractOrganization):
     uid = KpiUidField(uid_prefix='org')
 
+    @property
+    def email(self):
+        """
+        This exists to make dj-stripe happy
```

Comment:
While I appreciate a snarky comment when reading the code - the PR detail is the better place for this. Let's explain why this is needed. As I understand, djstripe needs this because Stripe requires an email on a Customer. Since our "customer" is an Organization - we need a means to get one and only one email for the Organization customer. You may want to review the original bug's stacktrace to ensure my understanding is correct.

Reply:
I understand why it's needed, I habitually wrote dj-stripe